### DISCUSSION TITLE
Want to batch process hundreds of DAISY titles that need unscrambling

### DISCUSSION THREAD

**Discussion by GeorgeKerscher** [daisy/pipeline#798] [Q&A]
While listening to discussions in Brazil, Francisco from ONCE said that they were unable to batch process DAISY titles that are scrambled. They would like to use the DAISY Pipeline to set up a process to batch these.

Yes, the DAISY Pipeline can unscramble one title at a time, but how can an organization set up batch processing?

Tom Mccartney did bach process of thefiles for the CSUN conference and he used the DAISY Pipeline. Of course, this is a command line interface (CLI) function.

Are there instructions on how to set this up anywhere?
@fmc@once.es

**Comment by torchtrust** [comment]
Dear George,
I appreciate it might be a bit old school, but it is effective, I export
from our database to a csv and then add the commands in new columns either
side of the variable data in a spreadsheet to build a command line and then
save it as a batch file, removing the tabs and run the batch file.
I haven't used it for this exact process but for validation of all our few
thousand DAISY books.
As you would expect the batch file just runs through the commands one at a
time until it finishes. Job done!
Hope that helps
kind regards
Paul

On Tue, 19 Nov 2024 at 20:57, George ***@***.***> wrote:

> While listening to discussions in Brazil, Francisco from ONCE said that
> they were unable to batch process DAISY titles that are scrambled. They
> would like to use the DAISY Pipeline to set up a process to batch these.
>
> Yes, the DAISY Pipeline can unscramble one title at a time, but how can an
> organization set up batch processing?
>
> Tom Mccartney did bach process of thefiles for the CSUN conference and he
> used the DAISY Pipeline. Of course, this is a command line interface (CLI)
> function.
>
> Are there instructions on how to set this up anywhere?
> @***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/daisy/pipeline/discussions/798>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AASSUUJWCHXV4XPQIQX7JQ32BOQ2LAVCNFSM6AAAAABSDBNISCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGUYTKNZUGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 

*Paul Wood*
Head of Technical Services
*Torch Trust*
Torch House, Torch Way,
Market Harborough, Leics. LE16 9HL, UK
Direct Line: *+44(0)1858 438269*
Mobile: *+44(0)7521 514212*
Tel: *+44(0)1858 438260*
Email: ***@***.***
Websites: www.torchtrust.org www.sightlossfriendlychurch.org.uk
<https://torchtrust.org/sight-loss-friendly-church>
Facebook: Torch-Trust
<https://www.facebook.com/pages/Torch-Trust/209414622516639>
Twitter: @TorchTrust <https://twitter.com/TorchTrust>
YouTube: Torch Trust on Video
<https://www.youtube.com/channel/UCjh3k2t6SckTWOtQZNmkN8w>

[image: Losing your sight? Know someone who is? Order a FREE audio Bible
player at torchtrust.org/pathway] <https://torchtrust.org/pathway>

[image: Registered with the Fundraising Regulator]
<https://www.fundraisingregulator.org.uk/>

Charity No. 1095904

Privileged/Confidential Information may be contained in this message.
If you are not the intended recipient please destroy this message
and kindly notify the sender by reply email. The computer from which
this mail originates is equipped with virus screening software.
However Torch Trust cannot guarantee that the mail and its attachments
are free from virus infection.

**Comment by marisademeglio** [comment]
Batch processing is now supported in the Pipeline app. I would suggest trying the newest beta, as the look and feel is improved.  The download is here:
https://github.com/daisy/pipeline-ui/releases/tag/1.9.0-beta
